psychopy/parallel/_inpout.py

```diff
@@ -35,7 +35,12 @@ def __init__(self, address=0x0378):
             self.base = int(address, 16)
         else:
             self.base = address
-        self.port = windll.inpout32
+        try:
+            hasattr(windll, 'inpout32')
```

Comment:
Since we now have the `hasattr()` calls both in `_inpout.py` and `parallel.py`, I am wondering whether it could be wise to change the signature of `_inpout.PParallelInpOut.__init__()`, and if `inpout` is `None` (default), we check for the "bitness" of the running Python install and choose `inpout32` or `inpoutx64` accordingly; `inpout` can also be a string (`inpout32` or `inpoutx64`) to explicitly select a driver.

```python
def __init__(self, address=0x0378, inpout=None):
    ...
    if inpout is None:
        # Check for bitness and select driver accordingly
        ...
    elif inpout == 'inpout32':
        self.port = windll.inpout32
    elif inpout == 'inpoutx64':
        self.port = windll. inpoutx64
    else:
        raise ValueError('Invalid driver specified')
    ...
```


In `parallel.py` we could then do:

```python
     try:
        from ._inpout import PParallelInpOut
        ParallelPort = PParallelInpOut
      except OSError:
          try:
              hasattr(windll, 'dlportio')
    ...
```

What do you think?